File: adaFront/src/app/services/color.service.ts
import { Injectable } from '@angular/core';
import { HttpClient } from '@angular/common/http';
import { URL_API } from './url.api';

@Injectable({
  providedIn: 'root'
})
export class ColorService {

  constructor(private http: HttpClient) {}

  findAllDropDown() {
    return this.http.get(`${URL_API}/api/color/dropdown`);
  }
}

File: adaApi/src/main/java/br/com/adaApi/api/service/ColorService.java
package br.com.adaApi.api.service;

import java.util.List;

import br.com.adaApi.api.dto.DropDown;
import br.com.adaApi.api.entity.Color;

public interface ColorService {

	List<Color> findAll();

	List<DropDown<Color>> findAllDropDown();

}
